tests/configs/.rr-kafka-tls.yaml

```diff
@@ -0,0 +1,49 @@
+version: '3'
+
+rpc:
+  listen: tcp://127.0.0.1:6002
+
+server:
+  command: "php php_test_files/jobs/jobs_ok.php"
+  relay: "pipes"
+  relay_timeout: "20s"
+
+logs:
+  level: debug
+  encoding: console
+  mode: development
+
+kafka:
+  brokers: ["127.0.0.1:9093"]
+  tls:
+    enabled: true
+    key: "test-certs/localhost+2-client-key.pem"
+    cert: "test-certs/localhost+2-client.pem"
+
+jobs:
+  num_pollers: 10
+  pipeline_size: 100000
+  timeout: 100
+  pool:
+    num_workers: 10
+    allocate_timeout: 60s
+    destroy_timeout: 60s
+
+  pipelines:
+    test-1-tls:
+      driver: kafka
+      config:
+        priority: 1
+        auto_create_topics_enable: true
+        producer_options:
+          max_message_bytes: 1000
+          required_acks: LeaderAck
+          compression_codec: snappy
+          disable_idempotent: true
+
+        consumer_options:
+          topics: [ "test-1-tls" ]
+          consumer_offset:
+            type: AtStart0
```

Comment:
Typo?

Reply:
What do you mean? In topic name?

Reply:
type: AtStart**0**, there is no such option.

Reply:
Oh, I didn't even think of that, I copied `.rr-kafka-init.yaml` where it is too :-)

Reply:
fixed